CHANGES.txt

```diff
@@ -143,6 +143,8 @@ Changes
 Fixes
 =====
 
+- Fixed ``EXPLAIN`` output for simple ``SELECT COUNT(*)`` queries.
```

Comment:
This isn't really a fix, but rather a regular improvement. It's documented that explain may not return detailed information. See https://github.com/crate/crate/blob/fc2219f6f256c34bb68599222d77adc66f362950/blackbox/docs/sql/statements/explain.txt#L31

Reply:
Ah ok, then I'll move it to Changes I guess?